packages/mobile/src/consumerIncentives/saga.test.ts

```diff
@@ -0,0 +1,140 @@
+import { toTransactionObject } from '@celo/connect'
+import { expectSaga } from 'redux-saga-test-plan'
+import { call } from 'redux-saga-test-plan/matchers'
+import { Actions as AlertActions } from 'src/alert/actions'
+import { ONE_CUSD_REWARD_RESPONSE } from 'src/consumerIncentives/ConsumerIncentivesHomeScreen.test'
+import { claimRewards, claimRewardsSuccess } from 'src/consumerIncentives/reducer'
+import { claimRewardsSaga } from 'src/consumerIncentives/saga'
+import { navigateHome } from 'src/navigator/NavigationService'
+import { Actions as TransactionActions } from 'src/transactions/actions'
+import { getContractKit } from 'src/web3/contracts'
+import { getConnectedUnlockedAccount } from 'src/web3/saga'
+import { getContract } from 'src/web3/utils'
+import { mockAccount, mockCeurAddress, mockCusdAddress } from 'test/values'
+
+const mockBaseNonce = 10
+const fundsSourceAddress = '0xfundsSource'
+
+jest.mock('src/web3/utils')
+jest.mock('@celo/connect')
+
+const contractKit = {
+  getWallet: jest.fn(),
+  getAccounts: jest.fn(),
+  web3: {
+    eth: {
+      getTransactionCount: jest.fn(() => mockBaseNonce),
+    },
+  },
+}
+
+const mockContract = {
+  methods: {
+    fundsSource: () => ({
+      call: () => fundsSourceAddress,
+    }),
+    claim: jest.fn(),
+  },
+}
+
+const mockTxo = {
+  sendAndWaitForReceipt: jest.fn(() => ({
+    transactionHash: '0x123',
+  })),
+}
+
+describe('claimRewardsSaga', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+    ;(toTransactionObject as jest.Mock).mockImplementation(() => mockTxo)
+  })
+
+  it('claiming no rewards succeeds', async () => {
```

Comment:
I think this isn't a possible case, right?

Reply:
It's not, we should never call it with an empty array, but when I started writing the test I had this and didn't see why to delete it 🤷‍♂️